modules/gitops-release-notes-1-11-4.adoc

```diff
@@ -0,0 +1,45 @@
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="fixed-issues-1-11-4_{context}"]
+== Fixed issues
+
+* Before this update, users could not use the `argocd-k8s-auth` binary to add GKE, and EKS clusters as this binary was not available in the GitOps container. This update fixes the issue as the `argocd-k8s-auth` binary has been added in the GitOps container. link:https://issues.redhat.com/browse/GITOPS-4226[GITOPS-4226]
+
+* Before this update, when Argo CD attempted to connect to Azure DevOps, an error was displayed due to the deprecation of the `rsa-ssh` host key algorithm, by the Azure DevOps Repository service. This update fixes the issue as Argo CD provides support for the `rsa-ssh` host key algorithms during the communication process between Argo CD and Azure DevOps Repository service. link:https://issues.redhat.com/browse/GITOPS-4543[GITOPS-4543]
+
+* Before this update, the `ignoreDifferences` sync option in Argo CD did not work for array fields. This update fixes the issue by modifying the merge strategy of the `ignoreDifferences` sync option used in the upstream project to handle array fields. As a result, the sync option now functions correctly by allowing users to ignore specific elements in the array during sync. link:https://issues.redhat.com/browse/GITOPS-2962[GITOPS-2962]
+
+* Before this update, users accessing the ROSA cluster after hibernation were unable to login to the Argo CD Web Console UI due to an error indicating an invalid redirect URI in the Dex configuration. To resolve this issue, users had to restart the {gitops-title} Operator pod along with the Server and Dex pods of Argo CD. With this update, users can now log in to the Argo CD Web Console without facing any errors once the ROSA cluster is operational post-hibernation. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
+
+* Before this update, users were unable to log in to the Argo CD Web Console UI when there is a delay in the availability of `openshift-gitops` route during processing of an instance of the Argo CD custom resource by the {gitops-title} Operator. An error message is displayed indicating an invalid redirect URI in the Dex configuration. To resolve this issue, users had to restart the {gitops-title} Operator pod along with the Server and Dex pods of Argo CD. With this update, users can now log in to the Argo CD Web Console without facing any errors. link:https://issues.redhat.com/browse/GITOPS-3736[GITOPS-3736]
+
+* Before this update, users could not create custom resources for Argo CD from the *Add* page on the *Developer* perspective of the {gitops-title} Web console. This issue has been observed from {gitops-title} 1.10 and later releases.
+link:https://issues.redhat.com/browse/GITOPS-4513[GITOPS-4513]
```

Comment:
The part of the fix is missing in this release note: "With this fix/this update fixes"
Is it fixed or is this rather a known issue?

Reply:
As per the information provided by the team, it has to be included in the Fixed issues section. I will ask for more inputs from the team to resolve this.

Reply:
Fixed.